systemd/system.service

```diff
@@ -1,6 +1,9 @@
 [Unit]
 Description=The Lounge (IRC client)
 After=network.target
```

Comment:
Shouldn't this be removed?

Reply:
Fair point, I would say yes, since network-online.target is a superset of network.target. 
I just did a reboot with the line removed and it worked as expected
I'm pushing the change

Reply:
@xPaw, I'm new to the world of github, I pushed the removal of the line in a new commit on the same branch I had used for the PR with "git push -u origin delayStart". I'm not sure if it's the proper way? I see the new commit 2d50c1e below this comment